packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -50,6 +51,42 @@ function buildAnswerMetadatum(fieldData: IJSONObject): IDataOutMetadatum {
   return answer
 }
 
+function buildAnswerArrayMetadatum(
+  fieldData: IJSONObject,
+): IDataOutMetadatum[] {
+  // there should only be checkbox and table fieldtypes that contain answer array
+  const answerArray = [] as IDataOutMetadata[]
```

Comment:
`IDataOutMetadatum[]`.

TBH `IDataOutMetadata` type checking is borked, I need to file a bug against TS